addons/web/static/src/core/dropdown/dropdown.js

```diff
@@ -110,6 +110,7 @@ export class Dropdown extends Component {
             popper: "menuRef",
             position,
             onPositioned: (el, { direction, variant }) => {
+                this.props.onPositioned({ direction, variant });
```

Comment:
check with boi if this props is necessary or if this can be handled in the dropdown directly